sdk/communication/azure-communication-administration/swagger/README.md

```diff
@@ -45,6 +48,15 @@ input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/257f060
 override-client-name: PhoneNumberAdminClient
 ```
 
+### Rename CommunicationIdentityAccessToken to CommunicationUserToken
```

Comment:
Move this to README.md inside `identity` SDK

Reply:
Deleted here.